CHANGELOG.rst

```diff
@@ -1,5 +1,42 @@
 Changelog
 =========
+1.19.1.dev0 (2021-08-25)
+------------------------
+MLflow 1.19.1.dev0 includes several major features and improvements:
+
+Features:
+
+- MLflow UI updates (#4712, @dbczumar)
```

Comment:
@dbczumar Are there any UI updates that should be highlighted?

Reply:
Yes @harupy:

```
- Improved scalability of the parallel coordinates plot for run performance comparison
- Added support for filtering runs based on their start time on the experiment page
- Added a dropdown for runs table column sorting on the experiment page
- Upgraded the AG Grid plugin, which is used for runs table loading on the experiment page, to version 25.0.0
- Fixed a bug on the experiment page that caused the metrics section of the runs table to collapse when selecting columns from other table sections
```